backend/certificates.js

```diff
@@ -0,0 +1,173 @@
+import {getSetting, saveSetting} from './settings.js';
+import * as fs from 'fs';
+import fetch from 'node-fetch';
+import Logger from './logger';
+
+import {fakeCerts} from '../test/backend/utils';
+
+const CA_HOST = {
+    PROTOCOL: 'http',
+    DOMAIN: 'plotly-connector-test.com',
+    ROUTE: 'certificate'
+};
+
+const daysCertificateIsGoodFor = 24;
```

Comment:
since this is just hardcoded, i'm assuming that the renewal isn't getting tested. could we make it a testing setting, drop it down to like 5 minutes, and add a renewal test?